app/assets/javascripts/app/views/notification_dropdown_view.js

```diff
@@ -87,6 +88,13 @@ app.views.NotificationDropdown = app.views.Base.extend({
     $(node).find(this.avatars.selector).error(this.avatars.fallback);
   },
 
+  onNotificationChange: function(notification) {
+    var node = this.dropdownNotifications.find("[data-guid=" + notification.get("id") + "]");
+    $(node).replaceWith(notification.get("note_html"));
+    $(node).find(".unread-toggle .entypo-eye").tooltip("destroy").tooltip();
```

Comment:
You can deduplicate these lines, because they are also used above in `onPushBack` and `onPushFront`.